README.md

```diff
@@ -36,6 +36,7 @@ The following environment variables can be passed in to customize run-time funct
 - HTTP_PORT - port used by http server (default 80)
 - DHCP_RANGE - dhcp range to use for provisioning (default 172.22.0.10-172.22.0.100)
 - MARIADB_PASSWORD - The database password
+- IRONIC_API_BEHIND_WSGI - Run ironic-api as a WSGI application. Can be true or false (default: true)
```

Comment:
Is there a reason to keep the old behavior around? It seems to me that using WSGI is always a win.

Reply:
Hi @dtantsur, thanks for your reviews.
I agree that using WSGI is a win.
Just wonder ifanyone may want to follow the legacy approach for some reasons? WDYT @maelk ?

Reply:
You still mention this option but it doesn't do anything any more?

Reply:
Done. Thanks